api/maven-api-meta/pom.xml

```diff
@@ -27,6 +27,7 @@
   </parent>
 
   <artifactId>maven-api-meta</artifactId>
-  <name>Maven API Meta annotations</name>
+  <name>Maven 4 API Meta annotations</name>
+  <description>Java 5 annotations for Maven 4 Immutable API.</description>
```

Comment:
I think the 5 is useless there days. Just "Java annotations" or "annotations" are good enough.